src/DOM/events/delegation.ts

```diff
@@ -19,6 +19,9 @@ export function handleEvent(name, lastEvent, nextEvent, dom) {
 		}
 		if (!lastEvent) {
 			delegatedRoots.count++;
+			if (name === 'onClick') {
```

Comment:
Ok, do you have a preference for OS detection method? Would something like `var iOS = !!navigator.platform && /iPad|iPhone|iPod/.test(navigator.platform);` suffice?

Referenced from: http://stackoverflow.com/a/9039885